checkstyle/checks/ParameterNumberCheck.hx

```diff
@@ -0,0 +1,44 @@
+package checkstyle.checks;
+
+import checkstyle.LintMessage.SeverityLevel;
+import haxeparser.Data;
+import haxe.macro.Expr;
+
+@name("ParameterNumber")
+@desc("Max number of parameters per method (default 7)")
+class ParameterNumberCheck extends Check {
+
+	public var severity:String = "INFO";
+	public var maxParameter:Int = 7;
```

Comment:
Same here. Please change `maxParameter` to `max` http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/sizes/ParameterNumberCheck.html